core/templates/pages/exploration-player-page/layout-directives/audio-bar.directive.ts

```diff
@@ -78,9 +81,11 @@ angular.module('oppia').directive('audioBar', [
 
           $scope.expandAudioBar = function() {
             $scope.audioBarIsExpanded = true;
+            AudioBarStatusService.expandAudioBar();
```

Comment:
Maybe `markAudioBarExpanded`, because the `expandAudioBar` seems like a function which which expand the audio-bar.